Working now

```diff
@@ -0,0 +1,5 @@
+Запускается через mvn test
+Работает выбор браузера через -Dbrowser.name=
+Работает выбор версии браузера через -Dbrowser.version= (Chrome работает, Firefox ругается)
+Работает выбор enviroment через -Denviroment= (если я правильно поняла, что имелось в виду). 
+Пока настроено через выбор ссылки стартовой страницы, в ближайшее время переделаю под разные Page.
```

Comment:
I am pretty sure  that it is easier to understand each other in Russian, but let's  try in English further... It will be useful for both of us